packages/x-components/src/components/result/__tests__/result-selector.spec.ts

```diff
@@ -0,0 +1,290 @@
+import Vue from 'vue';
+import { Wrapper, mount } from '@vue/test-utils';
+import { Result, ResultVariant } from '@empathyco/x-types';
+import ResultSelector from '../result-selector.vue';
+import { createResultStub } from '../../../__stubs__/results-stubs.factory';
+import { findTestDataById, getDataTestSelector } from '../../../__tests__/utils';
+import {
+  RESULT_KEY,
+  SELECTED_VARIANTS_KEY,
+  SET_RESULT_VARIANT_KEY
+} from '../../decorators/injection.consts';
+
```

Comment:
Since the `ResultSelector` is hard coupled to the `ResultProvider` I would test that using the `ResultProvider` here in this test.

Only testing `ResultSelector` features, but using the `ResultProvider` and not replicating its features here.